openshift/cronJob-template.yaml

```diff
@@ -0,0 +1,57 @@
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  annotations:
```

Comment:
I would add the usual annotations we have in all templates. Have a look at: https://github.com/AICoE/sesheta/blob/ede0ecaaf57b25ee14018db9d0a53defa6a53782/openshift/cronJob-template.yaml#L10